systemd/crc-self-sufficient-env.sh

```
@@ -1,8 +1,31 @@
+
+# set -o errexit disabled to capture the test return code
+set -o pipefail
+set -o nounset
+set -o errtrace
+
+TEST_TIMEOUT=120
+VSOCK_COMM_PORT=1024
+
+timeout "$TEST_TIMEOUT" python3 /usr/local/bin/crc-test-vsock.py "$VSOCK_COMM_PORT"
+
+case "$returncode" in
```

Shellcheck (0.11.0)

[warning] returncode is referenced but not assigned.

(SC2154)